### DISCUSSION TITLE
Key for Icons

### DISCUSSION THREAD

**Discussion by ph0rd** [openaudible/openaudible#701] [Q&A] [answered]
File list has nice icons -- (Audible logo, red dot, green dot, white dot, maybe more).
Where can I find what each icon means? 
Because as you can see in the screen capture, it is confusing and messy (multiple entries with different icons) ![Screenshot 2021-08-28 193114](https://user-images.githubusercontent.com/2100410/131236302-5de4ba3a-4e40-48d5-b73c-582cc4ab24f4.jpg)
I searched discussions, issues, openaudible.org - nothing.
Thanks in advance.

**Comment by openaudible** [comment, marked as answer]
Fixed in docs and application!

**Reply by openaudible** [reply]
Here is the section in the documentation:
https://openaudible.org/docs#Audiobook_Table
